source/manual/troubleshooting-vagrant.html.md

```diff
@@ -91,7 +91,7 @@ Using Cisco AnyConnect has been known to cause issues with NFS.
 
 Either disconnect from the VPN and reload the VM for access.
 
-Or, consider using [OpenConnect][] for your VPN. You can access the Aviation
+Or, consider using [OpenConnect](https://sites.google.com/a/digital.cabinet-office.gov.uk/gds/working-at-gds/how-to/connect-to-the-aviation-house-vpn) for your VPN. You can access the Aviation
```

Comment:
Oh ace - I had been wondering where that documentation is. Should that be migrated to these docs?

With that link added there's one in the footer to pull out.